token-swap/program/fuzz/src/instructions.rs

```diff
@@ -203,20 +205,26 @@ fn run_fuzz_instructions(fuzz_instructions: Vec<FuzzInstruction>) {
         );
     }
 
-    let pool_tokens = [&token_swap.pool_token_account, &token_swap.pool_fee_account]
-        .iter()
-        .map(|&x| get_token_balance(x))
-        .sum::<u64>() as u128;
+    // Omit fees intentionally, because fees in the form of pool tokens can
+    // dilute the value of the pool.  For example, if we perform a small swap
+    // whose fee is worth less than 1 pool token, we may round up to 1 pool
+    // token and mint it as the fee.  Depending on the size of the pool, this
+    // fee can actually reduce the value of pool tokens.
```

Comment:
nit: doesn't minting fees always decrease the value of pool tokens?

Reply:
It depends.  We're supposed to be withholding enough tokens from the trade to *not* reduce the value of pool tokens.  Ie. if we trade 10000 token A, 25 go as normal trading fees, and 5 go to the owner in the form of pool tokens.  If there's a clean conversion, ie. 5 tokens = 10 pool tokens, we're all good.  If 5 tokens = 1.6 pool tokens, and we mint 2 pool tokens, there will be value lost per token for that trade.

Reply:
Ah right, thanks for the explanation!